views/partials/user-list-single.ejs

```diff
@@ -0,0 +1,10 @@
+<tr>
+    <th scope="row"><%= user._id %></th>
```

Comment:
This should use `<td>` for semantic purposes.
it does not make sense to use the `TableHead` inside a `TableRow` if the `Rows` are not in the `Table`'s header element